app/src/controllers/filter/selectors.js

```diff
@@ -40,3 +40,21 @@ export const unsavedFilterIdsSelector = createSelector(
       })
       .map((item) => item.id),
 );
+
+export const dirtyFilterIdsSelector = createSelector(
+  launchFiltersSelector,
+  userFiltersSelector,
+  (filters, savedFilters) =>
+    filters
+      .map((filter) => ({
+        ...filter,
+        conditions: filter.conditions.filter(
+          (condition) => !(condition.filteringField === 'name' && condition.value === ''),
```

Comment:
What is this condition for?

Reply:
If you mean condition in line 52 - it is avoid filters with empty fields. This is a good question. it might happen that this condition will disable Discard button if we remove all data from name condition. Let me check this option. I will return back to you.

Reply:
I updated commit. That condition was redundant, I deleted it. I copied it from other selector. Still curious what was it doing.